### PR TITLE
Added 5 new questions

### PR DESCRIPTION
**Pull request by caitlind**
Added questions are:

- Celsius to farenheit (function)
- Farenheit to celsius (function)
- How many dogs? (debugging)
- Total evens (debugging)
- Rollercoaster ride (function and parsons)

As celsius and fahrenheit questions have been done in more than one branch we'll need to choose which ones to use.

### CODE REVIEW

**Comment by caitlind** [comment]
Would it be better if I added some read only lines to the debugging questions?

File: codewof/programming/content/en/celsius-to-farenheit/question.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Celsius to farenheit
```
Comment (lucyturn3r) [inline codewof/programming/content/en/celsius-to-farenheit/question.md]: Farenheit should be spelled fahrenheit

Comment (caitlind) [reply codewof/programming/content/en/celsius-to-farenheit/question.md]: Oops :P

File: codewof/programming/content/en/how-many-dogs/solution.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def number_of_dogs(pets):
+    total = 0
+    for i in range(len(pets)):
+        if pets[i].lower() == "dog":
+            total += 1
+        return total
```
Comment (lucyturn3r) [inline codewof/programming/content/en/how-many-dogs/solution.py]: I have a feeling that the return statement isn't meant to be indented here

File: codewof/programming/content/en/rollercoaster-ride/solution.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+def ride_the_rollercoaster(age, with_an_adult):
+    if age >= 10:
```
Comment (lucyturn3r) [inline codewof/programming/content/en/rollercoaster-ride/solution.py]: The question says they need to be 12 or older, but this condition says 10 or older

File: codewof/programming/content/en/how-many-dogs/question.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# How many dogs?
+
+The following function `number_of_dogs` takes a list `pets`, that is made up of strings, as an argument and **returns** the
+number of times the string `dog` or `Dog` appears in the list.
```
[the remark this answers is not in the document]
Comment (caitlind) [reply codewof/programming/content/en/how-many-dogs/question.md]: I can change it so it says it isn't case sensitive, does that sound good?

[the remark this answers is not in the document]
Comment (eAlasdair) [reply codewof/programming/content/en/how-many-dogs/question.md]: Much better

File: codewof/programming/content/en/celsius-to-fahrenheit/question.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Celsius to fahrenheit
+
+Write a function `celsius_to_fahrenheit(temperature)` that takes a temperature in celsius, and **returns** the temperature converted into fahrenheit.
+The formula for converting celsius into fahrenheit is: Celsius x 9/5 + 32 = Fahrenheit.
```
[the remark this answers is not in the document]
Comment (caitlind) [reply codewof/programming/content/en/celsius-to-fahrenheit/question.md]: Good point, that would read better

**Pull request merged**